### PR TITLE
Custom ner revamp

### PR DESCRIPTION
**Pull request by ramezgerges**
This PR introduces a major overhaul to the Custom NER powerskill, which is now updated to use the TextAnalytics SDK. The overhaul includes more detailed documentation, some testing, information and files for automated ARM deployment, and overall smaller and more robust code.

### CODE REVIEW

File: Text/CustomNER/README.md [hunk 1]
```diff
@@ -1,35 +1,81 @@
...
-6. Fill your Functions appsettings with the custom details from your deployment ('TA_ENDPOINT', 'TA_KEY', 'DEPLOYMENT', 'PROJECT_NAME' with the info you got in Language Studio after you deployed the model
-7. Add a field in your index where you will dump the enriched entities, more info [here](#sample-index-field-definition)
-8. Add the skill to your skillset as [described below](#sample-skillset-integration)
-9. Add the output field mapping in your indexer as [seen in the sample](#sample-indexer-output-field-mapping)
-10. Run the indexer 
+#### Deploy the powerskill to Azure
 
-## Sample Input:
+A powerskill is basically just an Azure Function written to be used as a custom skill in an Azure Cognitive Search pipeline. To deploy a function, an Azure App resource is needed. Notice the differnce between an Azure Function and an Azure App. If one is not already available, now is the time to create one. A good article that goes through all the steps of creating a simple Azure Function in Python is [Create a Python function using Visual Studio Code - Azure Functions | Microsoft Docs](https://docs.microsoft.com/en-us/azure/azure-functions/create-first-function-vs-code-python).
```
Comment (bleroy) [inline Text/CustomNER/README.md:57]: Typo: "differnce" -> "difference"

File: Text/CustomNER/README.md [hunk 2]
```diff
@@ -1,35 +1,81 @@
...
+
+```bash
+ zip -r customner-powerskill.zip custom_ner host.json requirements.txt
+```
+
+For more information on deployment methods, see [Deployment technologies in Azure Functions | Microsoft Docs](https://docs.microsoft.com/en-us/azure/azure-functions/functions-deployment-technologies).
+
+At this point, you should have a working Azure Function. Depending on how you deployed the function, you may need to add a `x-functions-key` header to each request to the function endpoint. The value for the header can be found in `Functions -> App keys` in the Function App resource page in Azure Portal.
+
+The function adhers to the input/output format specified by Azure Cognitive Search for custom skills. More information about custom skills and format, see [Custom skill interface - Azure Cognitive Search | Microsoft Docs](https://docs.microsoft.com/en-us/azure/search/cognitive-search-custom-skill-interface). Sample inputs and outputs for this powerskill are shown below. Notice that specifying the language is optional, and defaults to English.
```
Comment (bleroy) [inline Text/CustomNER/README.md:78]: typo: "adhers" -> "adheres"

**Review by bleroy — CHANGES_REQUESTED** [review]

File: Text/CustomNER/README.md (hunk 1 above)
Comment (ramezgerges) [reply Text/CustomNER/README.md]: Fixed in [a417a1b](https://github.com/Azure-Samples/azure-search-power-skills/pull/97/commits/a417a1b3c3c6dae7c6056b620329136fc4d4b12c)

File: Text/CustomNER/README.md (hunk 2 above)
Comment (ramezgerges) [reply Text/CustomNER/README.md]: Fixed in [a417a1b](https://github.com/Azure-Samples/azure-search-power-skills/pull/97/commits/a417a1b3c3c6dae7c6056b620329136fc4d4b12c)

**Review by bleroy — APPROVED** [review]

**Pull request merged**